docker/install/ubuntu_install_cmsis.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+set -e
+set -u
+set -o pipefail
+
+function show_usage() {
+    cat <<EOF
+Usage: docker/install/ubuntu_install_cmsis.sh <INSTALLATION_PATH>
+INSTALLATION_PATH is the installation path for the CMSIS.
+EOF
+}
+
+if [ "$#" -lt 1 -o "$1" == "--help" -o "$1" == "-h" ]; then
+    show_usage
+    exit -1
+fi
+
+INSTALLATION_PATH=$1
+shift
+
+CMSIS_VER="5.8.0"
+
+# Clone CMSIS
+git clone "https://github.com/ARM-software/CMSIS_5.git" "${INSTALLATION_PATH}"
```

Comment:
also suggest we do a shallow clone (`-depth=1 tags/${CMSIS_VER}`) while we're at it

Reply:
I think this doesn't work because `5.8.0` is not a branch and only a tag.

Reply:
what if we instead fetch a tar from the GH archives: `https://github.com/ARM-software/CMSIS_5/archive/5.8.0.tar.gz`?

Reply:
done. PTAL
thanks!